10HW/3_task_1.js

```diff
@@ -0,0 +1,66 @@
+// На сайте JSONPlaceholder - Free Fake REST API  с помощью функции fetch получить список пользователей. 
+// Вывести на экран: имя, e-mail, телефон и название компании пользователя.
+// Отдельными запросами получить список альбомов пользователя и список фотографий в альбомах. 
+// Дополнительно вывести список альбомов у пользователя с указанием количества в них фотографий. 
+// Для реализации трех запросов воспользоваться Promise.all().
+// (Принадлежность альбомов пользователем связано полем userId, принадлежность фотографий к альбомам сваязано полем albumId). 
+// Пример: 
+// 1.  name: Leanne Graham
+//     email: Sincere@april.biz
+//     phone: 1-770-736-8031 x56442
+//     company: Romaguera-Crona    
+//     albums:
+//       Album name 1 (10 photos)
+//       Album name 2 (100 photos)
+// __________________________________
+
+// 2.  name: Ervin Howell   
+//     email: Shanna@melissa.tv 
+//     phone: 010-692-6593 x09125
+//     company: Deckow-Crist
+//     albums:
+//       Album name 1 (10 photos)
+//       Album name 2 (100 photos)
+const URLs = {
+    baseUrl: `https://jsonplaceholder.typicode.com`,
+    users: (userID) => `/users/${userID}`,
+    userAlbums: (userID) => `/users/${userID}/albums`,
+    userPhotoInAlbums: (userID) => `/albums/${userID}/photos`,
+  };
+
+async function getUser(userId) {
+   return fetch(URLs.baseUrl + URLs.users(userId));
+};
+async function getALbums(userId) {
+    return fetch(URLs.baseUrl + URLs.userAlbums(userId));
```

Comment:
Почему сразу не возвращать данньіе, не нужно будет дополнительно мапить `response.json()` в фунции которая должна работать уже с полученой информацией
```suggestion
    const response = fetch(URLs.baseUrl + URLs.userAlbums(userId));
    const listOfAlbums = await response.json();
    return listOfAlbums;
```